src/_utils/TradeUtils.js

```diff
@@ -20,3 +20,50 @@ export const digitOptions = (from, to) => {
     const arr = Array.from(tArr);
     return arr.map((i, idx) => idx + from).map(x => ({ value: x, text: x.toString() }));
 };
+
+export const durationText = unit => {
+    switch (unit) {
+        case 't': return 'Ticks';
+        case 's': return 'Seconds';
+        case 'm': return 'Minutes';
+        case 'h': return 'Hours';
+        case 'd': return 'Days';
+        default: return null;
+    }
+};
+
+export const durationTypes = (min, max) => {
+    const minUnit = min.slice(-1)[0];
+    const maxUnit = max.slice(-1)[0];
+
+    const minIdx = durationUnit.indexOf(minUnit);
+    const maxIdx = durationUnit.indexOf(maxUnit);
+
+    return durationUnit.slice(minIdx, maxIdx + 1);
+};
+
+export const durationLarger = (a, b) => {
```

Comment:
These functions look like a good candidate for unit tests ;)
We start cautiously using them, and post-demo - very actively.